src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -674,6 +669,15 @@ public static boolean isClusterPerm(String action0) {
         );
     }
 
+    static boolean isDnfofAction(String action0) {
+        return (action0.startsWith("indices:data/read/")
+            || (action0.startsWith("indices:admin/mappings/fields/get")
+                || action0.equals("indices:admin/shards/search_shards")
+                || action0.equals("indices:admin/resolve/index"))
+            || action0.equals("indices:monitor/settings/get")
+            || action0.equals("indices:monitor/stats"));
+    }
```

Comment:
Not sure why there was nesting around some of those predicates, they all seem mutually exclusive

```suggestion
        return (action0.startsWith("indices:data/read/")
            || action0.startsWith("indices:admin/mappings/fields/get")
            || action0.equals("indices:admin/shards/search_shards")
            || action0.equals("indices:admin/resolve/index")
            || action0.equals("indices:monitor/settings/get")
            || action0.equals("indices:monitor/stats"));
    }
```

Reply:
Might have been a typo/spotless issue. I changed the implementation to wildcardMatcher in: [c12333c](https://github.com/opensearch-project/security/pull/3236/commits/c12333ce5e1442bdd206d23664a5b71e43787203), so I think this no longer applies.